package.json

```diff
@@ -9,8 +9,8 @@
     "optimizeImages": "npx cross-env VITE_TABLET=768 VITE_MOBILE=425 VITE_COMPRESS_QUALITY=80 node optimizeImages/optimizeImages.js",
     "stylelint": "npx stylelint \"**/*.scss\"",
     "stylelint:fix": "npx stylelint \"**/*.scss\" --fix",
-    "lint": "npx eslint . --report-unused-disable-directives --max-warnings 10",
-    "lint:fix": "npx eslint . --fix --report-unused-disable-directives --max-warnings 10",
+    "lint": "npx eslint . --report-unused-disable-directives --max-warnings 360",
```

Comment:
Let's increase it, e.g to 400, since there will be warnings added due to default exports.

Reply:
You will probably need to change the config so that there are no warnings
`export default [
    // ...other configurations
    stylistic.configs['recommended-flat'],
    {
        files: ["src/pages/*", ],
        rules: {
            'import/prefer-default-export': 'on',
        }
    },
    {
        files:['other files'],
        rules: {
            'import/prefer-default-export': 'off',
        }
    }
    // ...other configurations
];`

Reply:
default export warns already included in this number